tests/test_graphical_units/test_geometry.py

```diff
@@ -168,6 +168,40 @@ def test_Angle(scene):
     scene.add(a)
 
 
+@frames_comparison
+def test_three_points_Angle(scene):
```

Comment:
I think this test could be improved a bit: take a look at the rendered test output:

![image](https://user-images.githubusercontent.com/11851593/164542973-a66cec47-5a17-4574-bfde-cff133ec4fc6.png)

If there was a problem with any of the angles, it would be very hard to distinguish visually where the problem occurs. I suggest to `VGroup(acute, obtuse, q1, q2, q3, q4).arrange(RIGHT)` before adding all of the angles to the scene.

Reply:
That is a great suggestion! I just fixed that!